src/libteam/0013-teamd-lacp-port-admin-down-recv-not-processing.patch

```diff
@@ -0,0 +1,25 @@
+diff --git a/teamd/teamd_runner_lacp.c b/teamd/teamd_runner_lacp.c
+index 5fa026a..5f4ba75 100644
+--- a/teamd/teamd_runner_lacp.c
++++ b/teamd/teamd_runner_lacp.c
+@@ -1179,12 +1179,20 @@ static int lacpdu_recv(struct lacp_port *lacp_port)
+ 	struct lacpdu lacpdu;
+ 	struct sockaddr_ll ll_from;
+ 	int err;
++	bool admin_state;
+ 
+ 	err = teamd_recvfrom(lacp_port->sock, &lacpdu, sizeof(lacpdu), 0,
+ 			     (struct sockaddr *) &ll_from, sizeof(ll_from));
+ 	if (err <= 0)
+ 		return err;
+ 
++        admin_state = team_get_ifinfo_admin_state(lacp_port->ctx->ifinfo);
++        if (!admin_state) {
++                teamd_log_info("%s received PDU in admin down state.",
```

Comment:
I'd recommend to make it debug message. Otherwise we will create syslog message every 30 seconds on each member port of disabled port channel.

Reply:
Sure, the log message should be debug. I have changed it.
 
I will email libteam maintainer. But, as SONiC has more than 12 patches in libteam, it might consume significant time before the maintainer works/merges (or perhaps requests changes or provides comments on the proposed patches) these into libteam.

Reply:
Thanks for the fix. I'll check it later today.
Please email only your patch which fixing only one issue.

Reply:
Sure. Thanks.